modules/renter/contractor/consts.go

```diff
@@ -20,6 +20,14 @@ var (
 		Standard: 10,
 		Testing:  1,
 	}).(int)
+
+	// consecutiveRenewalsBeforeReplacement is the number of times a contract
+	// attempt to be renewed before it is marked as !goodForRenew.
+	consecutiveRenewalsBeforeReplacement = build.Select(build.Var{
+		Dev:      types.BlockHeight(12),
+		Standard: types.BlockHeight(12), // ~2h
+		Testing:  types.BlockHeight(12),
```

Comment:
Is 12 too high for the testing environments? A lot of our tests don't even have renew windows that long.

Reply:
I thought 12 is ok for testing since we probably don't want that to happen by accident since it might hide bugs in tests that don't explicitly want a failed renew to be replaced.